SonarRunner.Shim/SonarRunner.Shim.csproj

```diff
@@ -72,6 +72,7 @@
     <Compile Include="RoslynV1SarifFixer.cs" />
     <Compile Include="SonarProjectPropertiesValidator.cs" />
     <Compile Include="SonarRunner.Wrapper.cs" />
+    <None Include="packages.config" />
```

Comment:
The SonarRunner.Shim had a project reference to the Newtonsoft dll but didn't actually specify it as a package dependency. I've fixed that.